languages/php.php

```diff
@@ -17,7 +17,7 @@
 $string = 'Awesome cheatsheets';
 
 str_contains($string, 'cheat'); // Find if the string contains the specified string (PHP >= 8.0)
-str_replace('Awesome', 'Bonjour', $string); // Replace all occurence
+str_replace('Awesome', 'Bonjour', $string, $count); // Replace all occurence
```

Comment:
Both forms are valid, but we should leave the two lines and say that the `$count` variable is used to contain the number of replacements made.